s/x86_64.ss

```diff
@@ -3298,50 +3298,59 @@
                             [int* (list %rax %rdx)]
                             [fp* (list %Cfpretval %Cfparg2)]
                             [accum '()]
-                            [live* '()])
+                            [live* '()]
+                            [fp-live* '()])
                    (cond
                     [(null? result-classes)
                      (values (lambda ()
                                (if (pair? (cdr accum)) `(seq ,(car accum) ,(cadr accum)) (car accum)))
-                             live*)]
+                             live*
+                             fp-live*)]
                     [(eq? (car result-classes) 'integer)
                      (loop (cdr result-classes)
                            (fx+ offset 8)
                            (cdr int*)
                            fp*
                            (cons `(set! ,(car int*) ,(%mref ,%sp ,offset))
                                  accum)
-                           (cons (car int*) live*))]
+                           (cons (car int*) live*)
+                           fp-live*)]
                     [(eq? (car result-classes) 'sse)
                      (loop (cdr result-classes)
                            (fx+ offset 8)
                            int*
                            (cdr fp*)
                            (cons `(inline ,(make-info-loadfl (car fp*)) ,%load-double ,%sp ,%zero (immediate ,offset))
                                  accum)
-                           live*)]))]
+                           live*
+                           (cons (car fp*) fp-live*))]))]
                 [else
                  (values (lambda ()
                            ;; Return pointer that was filled; destination was the first argument
                            `(set! ,%Cretval ,(%mref ,%sp ,(if-feature windows (if adjust-active? 96 80) 48))))
-                         (list %Cretval))])]
+                         (list %Cretval)
+                         '())])]
               [(fp-double-float)
                (values
                 (lambda (x)
                   `(inline ,(make-info-loadfl %Cfpretval) ,%load-double ,x ,%zero ,(%constant flonum-data-disp)))
-                '())]
+                '()
+                (list %Cfpretval))]
               [(fp-single-float)
                (values
                 (lambda (x)
                   `(inline ,(make-info-loadfl %Cfpretval) ,%load-double->single ,x ,%zero ,(%constant flonum-data-disp)))
-                '())]
+                '()
+                (list %Cfpretval))]
               [(fp-void)
                (values (lambda () `(nop))
+                       '()
                        '())]
               [else
                (values(lambda (x)
```

Comment:
Add a space after values and indent the following lines to be consistent with the fp-void case.

Reply:
Also fixed - thanks!